conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git, Version
+from conan.tools.files import update_conandata, download, unzip
+
+required_conan_version = ">=2.0"
+
+class DPPConan(ConanFile):
+    name = "dpp"
+    version = "10.0.33"
+    default_user = "brainboxdotcc"
+    default_channel = "testing"
+    license = "https://github.com/brainboxdotcc/DPP/blob/master/LICENSE"
```

Comment:
If the license is not a official MIT/GPL/etc open-source license, maybe consider using the SPDX standard license specification, something like ``license = "DocumentRef-README:LicenseRef....``